aiomcache/client.py

```diff
@@ -247,6 +302,11 @@ async def _storage_command(self, conn: Connection, command: bytes, key: bytes,
         elif exptime < 0:
             raise ValidationException('exptime negative', exptime)
 
+        if flags == 0 and self._set_flag_handler and not isinstance(value, bytes):
+            value, flags = await self._set_flag_handler(value)
+        elif not isinstance(value, bytes):
+            raise ValidationException('value must be bytes if no flag handler')
```

Comment:
Hmm, do we need the `flags == 0` and set_flag_handler checks? If the value is not bytes, then surely we have set a handler, and the flags must not be set, otherwise the rest of the code will fail..

That would then allow us to simplify it to:

```suggestion
        if not isinstance(value, bytes):
            assert self._set_flag_handler
            value, flags = await self._set_flag_handler(value)
```

Otherwise, we could do something like:
```suggestion
        if not isinstance(value, bytes):
            if self._set_flag_handler is None:
                raise ValidationException("non-bytes value without set_flag_handler.")
            if flags != 0:
                raise ValidationException("non-0 flags with non-bytes value.")
            value, flags = await self._set_flag_handler(value)
```

Looking a little closer, currently `flags` is always set to 0 in all calls to _storage_command(). If we are deciding that the correct way to set flags is with this set_flag_handler, should we remove the flags parameter from this function? It looks to me like it was there in preparation for flags support, so if support doesn't involve that parameter, then I guess it's not needed.